homepage/homepage/app/(docs)/docs/[framework]/[[...slug]]/authentication/overview.mdx

```diff
@@ -6,12 +6,65 @@ import { CodeGroup, ContentByFramework } from "@/components/forMdx";
 
 Jazz authentication is based on cryptographic keys ("Account keys"). Their public part represents a user's identity, their secret part lets you act as that user.
 
-When a user loads a Jazz application for the first time, we create a new Account by generating keys and storing them locally.
+## Authentication States
 
-Without any additional steps the user can use Jazz normally, but they would be limited to use on only one device.
+Jazz has two distinct states for users without traditional sign-in: **Anonymous Authentication** and **Guest Mode**.
+
+### Anonymous Authentication
+
+When a user loads a Jazz application for the first time, we create a new Account by generating keys and storing them locally. This is **Anonymous Authentication**:
+
+- Users have full accounts with unique IDs
+- Data persists between sessions on the same device
+- Can be upgraded to a full account (passkey, passphrase, etc.)
+- Data syncs across the network (if enabled)
```

Comment:
I think we should also document what happens when the user register/authenticate with an auth provider.

Register -> The Anonymous account credentials are stored in the auth provider and the account is marked as Authenticated

Login -> The Anonymous account is discarded and the credentials are loaded from the auth provider. Data from the Anonymous account can be transferred using  `onAnonymousAccountDiscarded`

Reply:
I've added a new section, thanks